lib/taskana-core/src/test/java/acceptance/classification/CreateClassificationAccTest.java

```diff
@@ -290,4 +263,34 @@ void should_ThrowException_When_UserRoleIsNotAdminOrBusinessAdmin() {
 
     assertThatThrownBy(createClassificationCall).isInstanceOf(NotAuthorizedException.class);
   }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_SetDefaultServiceLevel_When_TryingToCreateClassificationWithMissingServiceLevel()
+      throws Exception {
+    Classification classification =
+        CLASSIFICATION_SERVICE.newClassification("newKey718", "", "TASK");
+    classification = CLASSIFICATION_SERVICE.createClassification(classification);
+    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+
+    classification = CLASSIFICATION_SERVICE.newClassification("newKey71", "", "TASK");
+    classification.setServiceLevel("");
+    classification = CLASSIFICATION_SERVICE.createClassification(classification);
+    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_SetDefaultServiceLevel_When_TryingToUpdateClassificationWithMissingServiceLevel()
+      throws Exception {
+    Classification classification =
+        CLASSIFICATION_SERVICE.getClassification("CLI:000000000000000000000000000000000001");
+    classification.setServiceLevel(null);
+    classification = CLASSIFICATION_SERVICE.updateClassification(classification);
+    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
+
+    classification.setServiceLevel("");
+    classification = CLASSIFICATION_SERVICE.updateClassification(classification);
+    assertThat(classification.getServiceLevel()).isEqualTo("P0D");
```

Comment:
separate into two tests? or a dynamic test?

Reply:
If so, I would prefer separated tests because those are only two cases. But I would suggest to leave it that way because of the small size and the conceptual similarity.